src/SFA.DAS.EmployerIncentives.Data/IncentiveApplication/IncentiveApplicationDataRepository.cs

```diff
@@ -39,8 +43,28 @@ public async Task Update(IncentiveApplicationModel incentiveApplication)
             if (existingApplication != null)
             {
                 _dbContext.Entry(existingApplication).CurrentValues.SetValues(model);
+                foreach (var apprenticeship in RemovedApprenticeships(existingApplication, model))
```

Comment:
still having to do this in EF Core makes me upset 😢

Reply:
Why removing them?  Are they not all being replaced?  It's just a case of saving the state of the model so you could just delete everything and replace it since you are just hydrating the model to the datastore for persistence.